<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SynapsePredictionProtocol</class>
 <widget class="QDialog" name="SynapsePredictionProtocol">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>582</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Synapse Prediction Protocol</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QLabel" name="siteListLabel">
       <property name="text">
        <string>Site list:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="siteListLabel2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Review: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="modeMenu"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="toReviewLabel">
       <property name="text">
        <string>Sites to be reviewed:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="firstButton">
       <property name="text">
        <string>First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="prevButton">
       <property name="text">
        <string>&lt;&lt; Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="reviewedLabel">
       <property name="text">
        <string>Finished sites:</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="reviewFirstButton">
       <property name="text">
        <string>First</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reviewPrevButton">
       <property name="text">
        <string>&lt;&lt; Prev</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="reviewNextButton">
       <property name="text">
        <string>Next &gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="currentLabel">
         <property name="text">
          <string>Current T-bar
</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="preLocationLabel">
         <property name="text">
          <string>--, --, --</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="preConfLabel">
         <property name="text">
          <string>Confidence: --</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="preStatusLabel">
         <property name="text">
          <string>Verified: --</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="gotoButton">
         <property name="text">
          <string>Go to current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="finishCurrentButton">
         <property name="text">
          <string>Finish current</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="lastVerifiedButton">
         <property name="text">
          <string>Last finished</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="progressLabel">
         <property name="text">
          <string>Progress:

--/-- (--%)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="postTableLabel">
         <property name="text">
          <string>PSDs (--/-- verified)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="sitesTableView">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="gridCheckBox">
       <property name="text">
        <string>Show grid</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>Refresh data</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>Exit protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="completeButton">
       <property name="text">
        <string>Complete protocol</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SynapsePredictionProtocol</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SynapsePredictionProtocol</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>577</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
